src/main/java/net/hockeyapp/android/objects/CrashManagerStatus.java

```diff
@@ -0,0 +1,30 @@
+package net.hockeyapp.android.objects;
+
+/**
+ * Crash Manager status
+ * @author Andreas Wörner
+ */
+public enum CrashManagerStatus {
```

Comment:
Yes, this is also taken from the iOS implementation, but not used for this feature. I will remove it from this pull-request, and provide a separate one.